data/json/items/gunmod/laser_gunmods.json

```diff
@@ -99,5 +100,97 @@
     "energy_drain_multiplier": 2.0,
     "min_skills": [ [ "weapon", 4 ], [ "electronics", 5 ] ]
+  },
+  {
+    "id": "add_intake_fan",
+    "type": "GUNMOD",
+    "name": { "str": "additional intake fan" },
+    "description": "An additional fan that mounts to the radiator intake. It slightly increases the cooling performance of the gun.",
+    "weight": "600 g",
+    "volume": "270 ml",
+    "integral_volume": "150 ml",
+    "price": 84000,
+    "price_postapoc": 1000,
+    "install_time": "20 s",
+    "material": [ "steel", "plastic" ],
+    "symbol": ":",
+    "color": "light_gray",
+    "location": "radiator",
+    "mod_targets": [ "pistol", "rifle" ],
+    "cooling_value_modifier": 0.5
+  },
+  {
+    "id": "heat_dissipation_sheets",
+    "type": "GUNMOD",
+    "name": { "str": "heat dissipation sheets" },
+    "description": "A micro-textured sheet that is applied to the heat exchange portion of the radiator. It increases cooling performance in a format that depends on the gun's original cooling performance.",
+    "weight": "150 g",
+    "volume": "600 ml",
+    "integral_volume": "0 ml",
+    "price": 84000,
+    "price_postapoc": 1000,
+    "install_time": "7 m",
+    "material": [ "steel" ],
+    "symbol": ":",
+    "color": "light_gray",
+    "location": "radiator",
+    "mod_targets": [ "pistol", "rifle" ],
+    "cooling_value_multiplier": 1.5
+  },
+  {
+    "id": "add_heat_sink_for_small",
+    "type": "GUNMOD",
+    "name": { "str": "additional heat sink for small" },
+    "description": "This is a common type additional heat sink in the form of thin metal plates are evenly spaced. It increases the heat capacity. Although it is designed for small guns, it does not mean that it cannot be attached to other guns.",
+    "weight": "300 g",
+    "volume": "200 ml",
+    "integral_volume": "0 ml",
+    "price": 84000,
+    "price_postapoc": 1000,
+    "install_time": "5 m",
+    "material": [ "steel" ],
+    "symbol": ":",
+    "color": "light_gray",
+    "location": "heatsink",
+    "mod_targets": [ "pistol", "rifle" ],
+    "overheat_threshold_modifier": 50
+  },
+  {
+    "id": "thermal_equilibrium_gel_sheet",
+    "type": "GUNMOD",
+    "name": { "str": "thermal equilibrium gel sheet" },
+    "description": "It is an additional part that wraps around the heating element of the gun body to improve thermal equilibrium efficiency. It does not increase the volume of the gun itself, but it cannot be installed together with a heat sink. It is effective when used with laser guns that originally have a large thermal capacity.",
+    "weight": "100 g",
+    "volume": "500 ml",
+    "integral_volume": "0 ml",
+    "price": 84000,
+    "price_postapoc": 1000,
+    "install_time": "10 m",
+    "material": [ "steel", "plastic" ],
+    "symbol": ":",
+    "color": "light_gray",
+    "location": "heatsink",
+    "mod_targets": [ "pistol", "rifle" ],
+    "overheat_threshold_multiplier": 1.5
+  },
+  {
+    "id": "power_save_emitter",
```

Comment:
I dont think reducing "heat_per_shot" is a reasonable gunmod effect for vanilla laser weapons. I see how mods might want the dial but vanilla lasers genereate so little heat in average that having this mod ends up as incredibly disruptive.

Reply:
The power_save_emitter, which is "heat_per_shot" -2, gives a not small numerical disadvantage of "damage_modifier": { "damage_type": "heat", "amount": -5 }.
The instantaneous damage amount can be calculated by overheat_threshold/heat_per_shot for a laser gun without recoil,
If we apply this to a vanilla gun, v29's heat_per_shot goes from 6 to 4, 66% of the original, but the damage is reduced to 5, 50% of the original.
The A7 laser rifle's heat_per_shot is reduced from 5 to 3, a 40% reduction, and the damage is reduced from 25 to 20, a 20% reduction. If this is shot until overheat, the firepower is increased by approximately 32%.
The heat_per_shot of the gatling_mech_laser is reduced from 6 to 4, the damage is reduced from 20 to 15, a 20% reduction, and at 200 with no additional heat sink, the original score of 680 becomes 750, an increase of about 10%.
In the case of recon_mech_laser_single, heat_per_shot is 17 and overheat_threshold is 100, so it only increases the number of shots that can be taken per shot.
Is it a big influence that the effect varies greatly depending on the gun to which it is attached and some of them have the opposite effect?

Of course I understand the danger of allowing heat_per_shot to simply be added or subtracted. I would not want to change the OVERHEAT system, because a minus 10 would easily remove most guns from the OVERHEAT system.
But if you want to change the amount of effect you can expect depending on the performance of the gun, I think this parameter is necessary. This is an adjustment that cannot be made with a percentage change.